src/plugins/vis_type_table/public/__snapshots__/table_vis_fn.test.ts.snap

```diff
@@ -2,12 +2,12 @@
 
 exports[`interpreter/functions#table returns an object with the correct structure 1`] = `
 Object {
-  "as": "visualization",
+  "as": "table_vis",
+  "params": Object {
+    "listenOnChange": true,
+  },
   "type": "render",
   "value": Object {
-    "params": Object {
-      "listenOnChange": true,
-    },
```

Comment:
@ananzh Does this structural change make sense to you? where `params` has moved up a level?

Reply:
@joshuarrrr thanks for checking this. I removed `listenOnChange` in the updated commit. I used UiState initially and `listenOnChange` is used to force re-render. Then to hookup to visBuilder, I switched to global state. test snap is also updated.